eoscompanion/main.py

```diff
@@ -177,4 +177,7 @@ def main(args=None):
                 os.pathsep.join([
                     GLib.getenv('XDG_DATA_DIRS') or ''
                 ] + flatpak_export_share_dirs), True)
+
+    logging.basicConfig(level=logging.DEBUG)
```

Comment:
What is logging configured as during the tests? (this only affects the main application).